docs/node-operators/get-started/setup-server.md

```diff
@@ -112,6 +112,15 @@ This will print the version of the binary you just built.
 
 :::note In order to run the `vega` command from any folder you should move the binary to a folder that is in your PATH.
 
+### Install Visor
+Visor manages protocol upgrades, allowing the nodes running a network to automatically update to the latest version of the Vega protocol, without requiring manual intervention. Using Visor is optional, but recommended. 
```

Comment:
"automatically update to the latest version" - > "automatically update to the latest agreed version" maybe??
"without requiring manual intervention" - > "without requiring manual intervention at the time of upgrade"